templates/puppet.conf.erb

```diff
@@ -47,3 +47,8 @@
 
     syslogfacility = <%= @syslogfacility %>
 <% end -%>
+<% if scope.lookupvar("puppet::server_directory_environments") -%>
+    environmentpath  = <%= scope.lookupvar("puppet::server_envs_dir") %>
+    basemodulepath   = <%= [scope.lookupvar("puppet::server_common_modules_path")].flatten.join(':') %>
+<% end -%>
+
```

Comment:
I've remove \n. If/When @domcleal approve it I'll squash the commits before merging